rocketpy/plots/monte_carlo_plots.py

```diff
@@ -175,8 +176,19 @@ def all(self, keys=None):
             raise ValueError("The 'keys' argument must be a string, list, or tuple.")
 
         for key in keys:
-            plt.figure()
-            plt.hist(self.monte_carlo.results[key])
-            plt.title(f"Histogram of {key}")
-            plt.ylabel("Number of Occurrences")
-            plt.show()
+            figure, plt = plt.subplots(3,1,sharex=True,gridspec_kw={'height_ratios':[1,3]})
+            
+            plt[0].boxplot(self.monte_carlo.results[key],vert=False)
+            plt[0].ytick([])
+
+            plt[1].hist(self.monte_carlo.results[key])
+            plt[1].title(f"Histogram of {key}")
+            plt[1].ylabel("Number of Occurrences")
+            
+            plt[2].hist(self.monte_carlo.results[key], density=True)
+            plt[2].title(f" Density {key}")
+            plt[2].ylabel("Probability Density")
+            kde = kde.gaussian_kde(self.monte_carlo.results[key])
+            x_array = np.linspace(min(self.monte_carlo.results[key]), max(self.monte_carlo.results[key]), 100) 
+            plt[2].plot(x_array, kde(x_array), label='KDE')
+            plt.show()
```

Comment:
This generates errors when I try to create plot `MonteCarlo` results. Indeed, you are replacing a global variable, the alias `plt` for the submodule `matplotlib.pyplot` for a local variable with the same name. Moreover, where does `kde` come from? It seems you have not imported it. Finally, I tried to do a quick clean of the errors just to get a gist of how it would look, and it seems that it creates **three different plots**. As mentioned in the PR, the idea was to create a **single** plot that blends the three (histogram, boxplot and density plot).